ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -36,7 +35,8 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.5.2') {
+//    implementation ('com.vungle:publisher-sdk-android:6.5.2') {
```

Comment:
why just don't change that line?

Reply:
Can simply change to GA build when uncomment this line.